workflow/sb_multicnv_wf.cwl

```diff
@@ -0,0 +1,331 @@
+class: Workflow
+cwlVersion: v1.2
+id: sb_multicnv_wf
+label: SB MultiCNV 1.0.0 Workflow
+doc: |-
+  In order to generate an automated solution which could work with an unlimited number of CNV callers or tumor samples, Seven Bridges has introduced additional changes to the SB Conseca CNV workflow logic. We have created a set of tools which can analyse combined results from different callers and present the best results needed for given analysis. 
+
+  **SB MultiCNV** consists of 3 parts: 
+  - **SB MultiCNV Combine**
+  - **SB MultiCNV Interpret**
+  - **SB MultiCNV Segment**
+
+
+
+  1. **SB MultiCNV Combine**
+
+  **SB MultiCNV Combine** tool is created in order to present the results from each caller. It divides regions in which callers are in disagreement and gathers results in the form of a .CSV file. In this way, a pool of data from different callers is created from which we can derive conclusions. Two types of input files should be provided on the input:
+  - Resulting CNV files obtained from different callers
+  - Information about the ploidy of the tumor sample. If this file is not provided, the ploidy parameter is set to 2.
+
+
+  2. **SB MultiCNV Interpret**
+
+  Second tool is **SB MultiCNV Interpret **which aims to explain the results obtained from **SB MultiCNV Combine** tool. Currently, there are two ways of interpretation:
+  - **Precise call** - Extracts regions where all callers provide the same status
+  - **Majority call** - Extracts regions where the majority of callers agree on the status. For each call which does not fit into any category a new status is defined: *ambiguous*.
+
+  3. **SB MultiCNV Segment**
+
+  After determining the final call status for each region, regions which have the same status are merged using the **SB MultiCNV Segment** tool.
+
+  ### **Input files**
+
+  **SB MultiCNV** takes on input both files with information about CNVs from caller and files with the information about the ploidy value. It is important for metadata to be set up, especially field: **Case ID** and **Sample type** (which gives information which tumor sample is being processed). If there are files with the same **Case ID** and **Sample type**, on file will be overwritten. 
+
+  Currently, SB MultiCNV can do the analysis of CNV output files from the following callers (with appropriate file extension):
+  - **PURPLE** ('.cnv.somatic.tsv')
+  - **GATK** ('.called.seg', '.modelFinal.seg')
+  - **PureCN** ('.seg', '_loh.csv')
+  - **CNVkit** ('.no_loh.cns', '.cns')
+  - **ControlFREEC** ('.value.txt')
+  - **VarSimLab** ('.CNV_results.txt')
+  - **CNVnator** ('calling_results.txt')
+  - **Sequenza** ('.segments.txt')
+  - **Facets** ('.cncf.tsv')
+  - **SBG Conseca CNV** ('.conseca.tsv')
+  - **Sclust** ('.allelic_states.txt'). 
+
+
+
+  Files with information about the ploidy value can be used only for specific callers:  
+    
+  - **PURPLE** ('.purple.purity.tsv')
+  - **ControlFREEC** ('.bam_info.txt')
```

Comment:
Update this to say `.info.txt` OR `.bam_info.txt` or not worth it?

Reply:
Will do. Should be quick.